frontend/compute-system/src/App.tsx

```diff
@@ -25,11 +24,17 @@ function App() {
           />
           <Route
             exact
-            path={routes.rating.path}
+            path={routes.appDetails.path}
             render={() => (
-              <AppRating
-                rate={mockRatings[0].rate}
-                description={mockRatings[0].description}
+              <AppDetails
+                id={1}
```

Comment:
I guess AppDetails should load this data itself so it doesn't have to have props. In the future it will get this data from Redux/React-sweet-state/api call.